src/views/default/done-expired.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+use dmstr\modules\contact\models\ContactLog;
+use dmstr\modules\contact\models\ContactTemplate;
+use dmstr\modules\prototype\widgets\TwigWidget;
+
+/**
+ * @var $schema string
+ * @var $model ContactLog
+ */
+?>
+
+<div class="container text-center">
+    <div class="row">
+    <div class=" col-xs-12 alert alert-success">
```

Comment:
Please use an extra div for the alert. Horizontal padding could lead to shifting layout.

Reply:
thanks, will change